packages/ui/src/components/BenefitCard/BenefitCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useState } from 'react';
+import { BoxProps, Center } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export interface BenefitCardProps extends BoxProps {
```

Comment:
🙃 As it's no longer a Box component it shouldn't receive `BoxProps` but `CenterProps`